core/server/data/migrations/versions/2.2/2-add-integrations-and-api-key-tables.js

```diff
@@ -0,0 +1,33 @@
+const commands = require('../../../schema').commands;
```

Comment:
How about extracting this like @allouis did in this [PR](https://github.com/TryGhost/Ghost/pull/9908)?
Then we can def merge this for Tuesday's release.

Reply:
Maybe we need a different convention for stacked PRs? Re-using WIP for something that's ready for review but has  dependency seems wrong if WIP PRs are ignored for review.

I try to always have a `requires xxx` line in the commit+PR but maybe it's not obvious enough.

Reply:
I am happy about with a new convention 👍 

> I try to always have a requires xxx

Yeah i have not seen this, wasn't an eye catcher. That's why i have commented on both PR's 💃 And confused myself.